src/OSPSuite.Core/Domain/Services/ModelConstructor.cs

```diff
@@ -116,9 +118,7 @@ public CreationResult CreateModelFrom(SimulationConfiguration simulationConfigur
       private void finalizeModel(IModel model, SimulationBuilder simulationBuilder)
       {
          _formulaTask.CheckFormulaOriginIn(model);
```

Comment:
Any chance that `CheckFormulaOriginIn` could somehow influence how `ExpandDynamicFormulaIn` works?